styles.css

```diff
@@ -1,56 +1,307 @@
-    color: #EDBCED;
-}
-
-
-/* Home area */
\ No newline at end of file
+        * {
+            box-sizing: border-box;
+            font-family: 'Montserrat', sans-serif;
+            font-weight: bold;
+            letter-spacing: .6px;
+            text-decoration: none;
+            text-transform: uppercase;
+        }
+        
+        body {
+            padding: 0;
+            margin: 0;
+        }
+        
+        .main-image-container {
+            background-image: url("image-assets/kazuend-19SC2oaVZW0-unsplash.jpg");
+            background-repeat: no-repeat;
+            background-size: cover;
+        }
+        /* Navbar */
+        
+        .navbar {
+            background-color: rgba(10, 42, 10, 0.6);
+            height: 15vh;
+            width: 100%;
+            border-bottom-left-radius: 25%;
+            border-bottom-right-radius: 25%;
+            position: fixed;
+            display: flex;
+            justify-content: center;
+            z-index: 1;
+        }
+        
+        .navbar-item-wrapper {
+            width: 70%;
+            display: flex;
+            justify-content: space-around;
+            align-items: center;
+        }
+        
+        .navbar a,
+        p {
+            color: #ECFAEC;
+            font-size: 1.5rem;
+        }
+        
+        .navbar a:active {
+            color: pink;
+        }
+        
+        .navbar a:hover {
+            color: #EDBCED;
+        }
+        /* Home area */
+        
+        .home {
+            height: 100vh;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+        }
+        
+        .home-container {
+            width: 60vh;
+            height: 60vh;
+            border-radius: 50%;
+            background-color: #ECFAEC;
+            border-color: #879908;
+            border-width: 10px;
+            border-style: groove;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+        }
+        
+        .center-circle {
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
+            overflow-wrap: normal;
+            font-size: 2rem;
+            padding: 20%;
+        }
+        
+        .circle-inner-wrapper {
+            width: 75%;
+            height: 75%;
+            text-align: center;
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
+        }
+        
+        .circle-inner {
+            padding: 5%;
+        }
+        
+        .hex-icons {
+            font-size: 3rem;
+            color: #ECFAEC;
+        }
+        
+        .hex-icons:hover {
+            color: #EDBCED;
+        }
+        /* About me */
+        
+        .about-me-navbar-bumper-container {
+            background-image: url(image-assets/forest-3742949.jpg);
```

Comment:
The background image should really be in quotes, it will work without them but newer coding standards call for quotes.